components/alert/src/alert.tsx

```diff
@@ -7,11 +7,11 @@ import type { VariantProps } from "../../types";
 const alertVariants = cva("relative w-full rounded-lg border px-4 py-3 text-sm flex gap-4", {
 	variants: {
 		priority: {
-			danger: "border-red-600 bg-red-500/5 text-red-700",
-			default: "border-gray-600 bg-gray-500/3 text-gray-700",
-			info: "border-blue-600 bg-blue-500/5 text-blue-700",
-			success: "border-green-600 bg-green-500/5 text-green-700",
-			warning: "border-amber-600 bg-amber-500/5 text-amber-700",
+			danger: "border-danger-muted bg-danger-muted text-bg-danger-muted",
```

Comment:
Just noting that these colors have moved around a bit from what was originally spec'd. Just a little less contrast, but I think it's passable 🫡

Reply:
Yeah, I softened the border a bit. We don't have to do it, but I did feel they were a tad florescent.